packages/vc-verification/src/issuer-verification.ts

```diff
@@ -0,0 +1,230 @@
+import * as jwt from 'jsonwebtoken';
+import { providers } from 'ethers';
+import { ProofVerifier } from '@ew-did-registry/claims';
+import { Resolver } from '@ew-did-registry/did-ethr-resolver';
+import { RegistrySettings } from '@ew-did-registry/did-resolver-interface';
+import { CredentialResolver, IssuerDefinitionResolver } from '.';
+import {
+  IVerifiableCredential,
+  VerificationResult,
+  OffChainClaim,
+} from './models';
+import { upgradeChainId } from './upgrade-chainid';
+
+export class IssuerVerification {
+  private _provider: providers.Provider;
+  private _resolver: Resolver;
+  private _registrySetting: RegistrySettings;
```

Comment:
Also doesn't need to be an instance variable as it is just used to instantiate `_resolver`